Olympus2.hebontes/cba_settings.sqf

```diff
@@ -4,3 +4,4 @@ force tac_apollo_enabled = false;           // Set to 'false' to disable persist
 // ACE Settings
 force ace_weather_windSimulation = false;   // Wind based on the maps geographical location. Enabling this will override the settings you've set in EDEN.
 force ace_medical_enableFor = 1;            // Enables ACE Advanced Medical for AI
+force ace_viewdistance_viewDistanceOnFoot = 5; // Performance saving when on Ground ranges
```

Comment:
What? This forces it to 5km for everyone? I have it set to 2km myself, so this is more?

Why does this have to be done at all? If you want to reduce it to some max range for ranges specifically, then overwrite it on range start and reset on range end.

Reply:
It forces it to 2.5KM, And it saves us time with everyone coming on complaining of low FPS while running with 12KM view distance on.

Reply:
I mean, those people are going to have bad performance on all non-CUP terrains.

What happens when someone wants to play around with snipers etc. and they can't even see the target?

Comment is still false, that's not just for ranges, it's for anything on foot, forced.

Reply:
Fair enough, i'll remove the setting.

Reply:
Setting is not the way to go, but you can force set it on range start and reset it on range end. `(get/set)ObjectViewDistance` / `(get/set)ViewDistance`, shouldn't even break with ACE3 View Distance unless someone changes it during a range, and even then it would just overwrite the custom one.